uat/README.md

```diff
@@ -0,0 +1,103 @@
+# End-to-end tests for Greengrass Development Kit (GDK) - Command Line Interface (CLI)
+
+### *Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.*
+
+End-to-end setup uses `pytest` and expects test dependencies to be installed.
+
+### Getting Started
+
+#### Prerequisites
+1. [Python3](https://www.python.org/downloads/) and [pip](https://pip.pypa.io/en/latest/installation/): As the GDK CLI tool is written in python, you need to have python3 and pip installed. The most recent version of python includes pip.
+
+2. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html): As you'd have to configure your AWS credentials using AWS CLI before running certain gdk commands.
+
+#### 1. Cloning CLI repository
+
+To install the latest version of CLI using this git repository and pip, run the following command
+
+```shell
+git clone https://github.com/aws-greengrass/aws-greengrass-gdk-cli.git@v1.1.0
+cd aws-greengrass-gdk-cli
+```
+
+#### 2. Install CLI from code
+
+```shell
+python3 -m pip install pip
+pip install .
+```
+
+#### 3. Configure AWS credentials
+
+Configure AWS CLI with your credentials as shown here - https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html.
+
+#### 4. Installing test dependencies
```

Comment:
IMO, this can be moved to a separate README inside github workflows or uat folder. This is the landing page for customers and it has more information than needed to install and work with gdk. :)